remoteworkitem/scheduler.go

```diff
@@ -13,13 +13,14 @@ import (
 )
 
 // TrackerSchedule capture all configuration
-type trackerSchedule struct {
-	TrackerID   uuid.UUID
-	URL         string
-	TrackerType string
-	Query       string
-	Schedule    string
-	SpaceID     uuid.UUID
+type TrackerSchedule struct {
```

Comment:
Why does this struct type need to be exported? Is it used outside of the `remoteworkitem` package?

Reply:
yes - https://github.com/fabric8-services/fabric8-wit/pull/2360/files#diff-1f33ef79fe402cddeafe4b0125b556bcR51